<launch>
	<arg name="bag_name" default="2020-09-14-18-01-31.bag" />
	<arg name="bag_file" default="/home/mateus/rosbags/$(arg bag_name)" />
	<arg name="topics" default="/d435i/color/camera_info /d435i/color/image_raw /d435i/aligned_depth_to_color/image_raw /d435i/imu /vins_node/odometry /vins_node/path"/>
	<!--arg name="topics" default="/d435i/color/camera_info /d435i/color/image_raw /d435i/imu /vins_node/odometry /vins_node/path"/-->
	<arg name="output_bag" default="$(find bag_from_slam)/outputs/from_slam_$(arg bag_name)"/>

	<node pkg="rosbag" type="play" name="player" required="true" output="screen" args="--clock $(arg bag_file)"/>

	<!--include file="$(find vins)/launch/vins_rviz.launch" /-->

	<node pkg="vins" type="vins_node" name="vins_node" args="/home/mateus/catkin_ws/src/VINS-Fusion/config/realsense_d435i/realsense_stereo_imu_config.yaml"/>

	<node pkg="rosbag" type="record" name="recorder" args="record -O $(arg output_bag) $(arg topics)"/>
</launch>
